### PR TITLE
Add identity from mnemonic and clean code

### PR DESCRIPTION
**Pull request by mircial**

### CODE REVIEW

File: ic/agent.py [hunk 1]
```diff
@@ -113,6 +99,11 @@ def read_state_raw(self, canister_id, paths):
         }
         _, data = sign_request(req, self.identity)
         ret = self.read_state_endpoint(canister_id, data)
+        if ret == b'Invalid path requested.':
+            raise ValueError('Invalid path requested!')
+        elif ret == b'Could not parse body as read request: invalid type: byte array, expected a sequence':
+            raise ValueError('Could not parse body as read request: invalid type: byte array, expected a sequence')
+        print(ret)
```
Comment (Myse1f) [inline ic/agent.py:106]: maybe remove this debug print

Comment (mircial) [reply ic/agent.py:106]: if user input wrong paths, those error info could prompt them

Comment (Myse1f) [reply ic/agent.py:106]: I mean the `print` in line 106

Comment (mircial) [reply ic/agent.py:106]: yeah, I delete it now.

**Pull request merged**